recipes/2.0/variant_detection/environment-2.0.yml

```diff
@@ -0,0 +1,35 @@
+name: gencore_variant_detection
```

Comment:
If we are going to have gencore_align then there is no need to have aligners in this module (such as tophat bwa bowtie etc.)